README.md

```diff
@@ -87,27 +87,39 @@ from lapixdl.evaluation.visualize import show_detections
 The available color maps are the [ones from matplotlib](https://matplotlib.org/3.1.1/gallery/color/colormap_reference.html).
 
 ### For Data Conversion
-
-This module exports the following functions for data conversion:
-```python
-from lapixdl.convert.labelbox import labelbox_to_coco
-```
+This module exports the functions for data conversio.
```

Comment:
Why did you remove the list of exported functions?

Reply:
Because I change the way of the converter works, and don't know if I like the part of have the function `create_COCO_OD`... and I forgot to add it to the readme